docs/examples.rst

```diff
@@ -466,6 +466,21 @@ I guess that means Clojure can be shut down now, sorry Rich!
    >>> i1 == i2
    False
 
+If you're still not convinced, that Python + ``attrs`` is the better Clojure, maybe immutable classes can change your mind:
+
+.. doctest::
+
+   >>> @attr.s(frozen=True)
```

Comment:
Since things are not quite immutable in Python, have you considered `frozen='ish'`. It's only an extra character ;)